sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_dsl_pipeline.py

```diff
@@ -2648,8 +2653,8 @@ def register_both_output():
         assert pipeline_output.name == "b_output"
         assert pipeline_output.version == "2"
         node_output = pipeline_job.jobs["node"].outputs.component_out_path
-        assert node_output.name == "a_output"
-        assert node_output.version == "1"
+        assert node_output.name == None
+        assert node_output.version == None
```

Comment:
Do we have test when pipeline level don't have output name, version setting and check node level outputs can be promoted to pipeline level?

Reply:
Yes, in `test_register_with_output_format`, we don't set name and version for pipeline output `pipeline_b_output` and we assert the name and version of this output.